testing/start.rst

```diff
@@ -0,0 +1,64 @@
+How to test a Plone product
+===========================
+
+To better understand how to test a Plone product, the best thing is to create a new Plone package from scratch
+and use it to see different testing techniques.
+
+We are going to use `plonecli <https://pypi.org/project/plonecli/>`_ tool heavily because it allows to easily create new Plone package
+and features (content-types, views, vocabularies) with simple commands.
+
+Create Package
+--------------
+
+First of all we need to create a virtual Python environment:
+
+.. code-block:: console
+
+  $ virtualenv .env
```

Comment:
Why don't you follow the instructions at https://github.com/plone/plonecli ?
I would expect some confusion if the two documents differ.
If there's a good reason to do it different here, explain why.

Reply:
Documentation says to install plonecli in global user site-packages..is it the recommended way to install and use plonecli? If yes, i could use the same instructions

Reply:
Yes, afaik it is. Personally I use pyenv and do it differently, which is more flexible for me, but OTOH to complex for newcomers.

Reply:
done!

Reply:
pip install plonecli --user makes sense, because you don't want to install plonecli all the time but you don't want to mess up the system global Python either.